easybuild/easyconfigs/p/Perl/Perl-5.34.0-GCCcore-11.2.0.eb

```
@@ -1804,6 +1804,16 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MARKOV/'],
         'checksums': ['b2658b53176df5afa5d02789368715c86b98c8d04ecd930252bcd7f832cc6224'],
     }),
+    ('Sys::Info::Driver::Unknown', '0.79', {
+        'source_tmpl': 'Sys-Info-Driver-Unknown-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BU/BURAK/'],
+        'checksums': ['02408843c8e36ea3d507e9f33fee48d6908543829ebe320f13d1bfe76af31e09'],
+    }),
+    ('Sys::Info::Driver::Linux', '0.7905', {
+        'source_tmpl': 'Sys-Info-Driver-Linux-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BU/BURAK/'],
+        'checksums': ['899c329bd3508ec5849ad0e5dadfa7c3679bbacaea9dda12404a7893032e8b7b'],
+    }),
 ]
```

```suggestion
    ('Unix::Processors', '2.046', {
        'source_tmpl': 'Unix-Processors-%(version)s.tar.gz',
        'source_urls': ['https://cpan.metacpan.org/authors/id/W/WS/WSNYDER/'],
        'checksums': ['3973ebdc44682c9c15c776f66e8be242cb4ff1dd52caf43ff446b74d4dccca06'],
    }),
]
```

Sorry - one more needed for #15191